openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -528,6 +528,14 @@ def _get_representations(self, instance, exp_files):
             # preview video rendering
             for app in self.aov_filter.keys():
                 if os.environ.get("AVALON_APP", "") == app:
+                    # no need to add review if baking in nuke present
+                    if (
+                        app == "nuke"
```

Comment:
I would check here for more general key on instance data like `"addPreviewOnSubmit"` which would be set in Nuke's submitter or other plugin, to avoid host specific code in general plugin.